recipes/zopfli/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ZopfliConan(ConanFile):
+    name = "zopfli"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/zopfli/"
+    description = "Zopfli Compression Algorithm is a compression library programmed in C to perform very good, but slow, deflate or zlib compression."
+    topics = ("conan", )
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake",
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        cmake = CMake(self)
+        cmake.definitions['ZOPFLI_BUILD_INSTALL'] = True
+        cmake.configure()
+        self._cmake = cmake
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "ZopFli"
+        self.cpp_info.names["cmake_find_package_multi"] = "ZopFli"
+
+        self.cpp_info.components["libzopfli"].names["cmake_find_package"] = "libzopfli"
+        self.cpp_info.components["libzopfli"].names["cmake_find_package_multi"] = "libzopfli"
+        self.cpp_info.components["libzopfli"].libs = ["zopfli"]
```

Comment:
```suggestion
        self.cpp_info.components["libzopfli"].libs = ["zopfli"]
        if self.settings.os == "Linux":
            self.cpp_info.components["libzopfli"].system_libs = ["m"]
```

https://github.com/google/zopfli/blob/zopfli-1.0.3/CMakeLists.txt#L99

Reply:
weird, it links without libm